From: Mike Hommey <mh@glandium.org>
Date: Mon, 16 Jun 2008 10:41:05 +0200
Subject: Set MOZ_APP_NAME to iceweasel

This will make the shell service use the proper path when setting the
default browser.
---
 browser/confvars.sh |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/browser/confvars.sh b/browser/confvars.sh
index 7173d2c..18aae5c 100755
--- a/browser/confvars.sh
+++ b/browser/confvars.sh
@@ -36,7 +36,7 @@
 #
 # ***** END LICENSE BLOCK *****
 
-MOZ_APP_NAME=firefox
+MOZ_APP_NAME=iceweasel
 MOZ_UPDATER=1
 MOZ_PHOENIX=1
 
